tests/compartmentReport.cpp

```diff
@@ -377,8 +377,8 @@ BOOST_AUTO_TEST_CASE( test_convert_and_compare )
     uris.push_back( brion::URI( temp.string() + ".h5" ));
     uris.push_back( brion::URI( std::string( "leveldb:///" ) + random ));
     uris.push_back( brion::URI( std::string( "leveldb:///o" ) + random ));
-    uris.push_back( brion::URI( std::string( "memcached:///" ) + random ));
-    uris.push_back( brion::URI( std::string( "memcached:///o" ) + random ));
+    // uris.push_back( brion::URI( std::string( "memcached:///" ) + random ));
+    // uris.push_back( brion::URI( std::string( "memcached:///o" ) + random ));
```

Comment:
Write a comment explaining why you're commenting this code, otherwise someone else could remove it later thinking it's dead code.